app/views/handlers/individual_response.py

```diff
@@ -73,24 +73,34 @@ def __init__(
         self._list_name = self._schema.get_individual_response_list()
 
         self._metadata = self._questionnaire_store.metadata
-        self.page_title = None
 
         if not self._is_location_valid():
             raise NotFound
 
-    def _is_location_valid(self):
-        self._list_name = self._schema.get_individual_response_list()
+    @cached_property
+    def _list_model(self):
+        return self._questionnaire_store.list_store[self._list_name]
 
-        list_model = self._questionnaire_store.list_store[self._list_name]
+    @cached_property
+    def _list_item_position(self):
+        return self._list_model.index(self._list_item_id)
+
+    def get_full_page_title(self, page_title):
+        if self._list_item_id:
+            page_title += ": Person {list_item_position}".format(
```

Comment:
The separator (`: `) should be separate from the rest of the string, and `Person {list_item_position}` needs to be wrapped in a `lazy_gettext` call.

Reply:
Done in 8e5d7c91